### DISCUSSION TITLE
Wilderness revert on mob explosion overriding blocks.

### DISCUSSION THREAD

**Discussion by Milireso** [TownyAdvanced/Towny#7246] [Q&A]
Is there a way to stop the regeneration features of the plugin from replacing blocks, especially valuable ones placed by players after the explosion? I know the chance of this happening during normal gameplay is relatively low, but it could lead to item loss. 
Currently, if a player places an ore or a chest inside the crater, it will be deleted and replaced with dirt, stone, or other block of much lesser value.
I have two ideas that could fix the problem. 
The first one would be adding chests to block_ignore (line 508) and checking if the block is not one of the valuables (or better if it's air) before regenerating. 
The second one - displaying a warning message to a player attempting to fix creeper holes and delaying it for a minute or canceling.

**Comment by LlmDl** [comment]
I have never encountered a player that sees a creeper blow up and then they immediately jump in to put their valuable blocks into the hole.

Towny does not have a feature that saves blocks from being regenerated over top of. Your players should have common sense.
